pkg/operation/botanist/component/kubestatemetrics/monitoring.go

```diff
@@ -0,0 +1,342 @@
+	monitoringMetricKubePodLabels                                                                   = "kube_pod_labels"
+	monitoringMetricKubePodStatusPhase                                                              = "kube_pod_status_phase"
+	monitoringMetricKubePodStatusReady                                                              = "kube_pod_status_ready"
+	monitoringMetricKubeReplicaSetMetadataGeneration                                                = "kube_replicaset_metadata_generation"
+	monitoringMetricKubeReplicaSetOwner                                                             = "kube_replicaset_owner"
+	monitoringMetricKubeReplicaSetSpecReplicas                                                      = "kube_replicaset_spec_replicas"
+	monitoringMetricKubeReplicaSetStatusObservedGeneration                                          = "kube_replicaset_status_observed_generation"
+	monitoringMetricKubeReplicaSetStatusReplicas                                                    = "kube_replicaset_status_replicas"
+	monitoringMetricKubeReplicaSetStatusReadyReplicas                                               = "kube_replicaset_status_ready_replicas"
+	monitoringMetricKubeStatefulSetMetadataGeneration                                               = "kube_statefulset_metadata_generation"
+	monitoringMetricKubeStatefulSetReplicas                                                         = "kube_statefulset_replicas"
+	monitoringMetricKubeStatefulSetStatusObservedGeneration                                         = "kube_statefulset_status_observed_generation"
+	monitoringMetricKubeStatefulSetStatusReplicas                                                   = "kube_statefulset_status_replicas"
+	monitoringMetricKubeStatefulSetStatusReplicasCurrent                                            = "kube_statefulset_status_replicas_current"
+	monitoringMetricKubeStatefulSetStatusReplicasReady                                              = "kube_statefulset_status_replicas_ready"
+	monitoringMetricKubeStatefulSetStatusReplicasUpdated                                            = "kube_statefulset_status_replicas_updated"
+	monitoringMetricKubeVerticalPodAutoscalerStatusRecommendationContainerRecommendationsTarget     = "kube_verticalpodautoscaler_status_recommendation_containerrecommendations_target"
+	monitoringMetricKubeVerticalPodAutoscalerStatusRecommendationContainerRecommendationsUpperBound = "kube_verticalpodautoscaler_status_recommendation_containerrecommendations_upperbound"
+	monitoringMetricKubeVerticalPodAutoscalerStatusRecommendationContainerRecommendationsLowerBound = "kube_verticalpodautoscaler_status_recommendation_containerrecommendations_lowerbound"
+	monitoringMetricKubeVerticalPodAutoscalerSpecUpdatePolicyUpdateMode                             = "kube_verticalpodautoscaler_spec_updatepolicy_updatemode"
+)
+
+var (
+	centralMonitoringAllowedMetrics = []string{
+		monitoringMetricKubePersistentVolumeClaimResourceRequestsStorageBytes,
```

Comment:
Nit: While we were diffing the old and new config we noticed that this metrics is now the first one in the regex while previously it was in the middle of the list? And it looks like the metrics order follows the alphabetical order, isn't it? Maybe we can again place it in its old order.